simd.dd

```diff
@@ -57,9 +57,25 @@ import core.simd;
 ---
 
 	$(P These types and operations will be the ones defined for the architecture
-	the compiler is targetting. If a particular CPU family has varying
+	the compiler is targeting. If a particular CPU family has varying
 	support for vector types, an additional runtime check may be necessary.
-	The compiler does not emit runtime checks; those must be done by the programmer.
+	The compiler does not emit runtime checks; those must be done by the 
+	programmer.
+	)
+	
+	$(P It is possible to test if the architecture the compiler is targeting
+	supports SIMD types by using:)
+---
+version(D_SIMD)
```

Comment:
This is not true, as I've mentioned time and again.

When SIMD support was introduced, I made sure that ``version(D_SIMD)`` only applied to DMD's vendor-specific ``__simd`` intrinsic.